docs/requirements/TestRequirements.md

```diff
@@ -48,7 +48,7 @@ Tests to implement
 |R16|NS authoritative answer                                 |[DELEGATION](../specifications/tests/Delegation-TP/delegation04.md)|
 |R17|NS name has a valid domain/hostname syntax              |[SYNTAX](../specifications/tests/Syntax-TP/syntax04.md)|
 |R18|NS is not an alias                                      |[DELEGATION](../specifications/tests/Delegation-TP/delegation05.md)|
-|R19|NS can be resolved                                      |[DELEGATION](../specifications/tests/Delegation-TP/??)|
+|R19|NS can be resolved 				     |[DELEGATION](../specifications/tests/Delegation-TP/delegation01.md)|
```

Comment:
I think it should be "../specifications/tests/Nameserver-TP/nameserver06.md"

``
Test case identifier

NAMESERVER06 NS can be resolved
``